packages/utils/src/casing.ts

```diff
@@ -33,33 +33,30 @@ export function snakelize<T>(object: T): Snakelize<T> {
 }
 
 export function snakeToCamelCase(str: string): string {
-  if (!str.includes('_')) return str
-
-  let result = ''
-  for (let i = 0, len = str.length; i < len; ++i) {
-    if (str[i] === '_') {
-      result += str[++i].toUpperCase()
-
-      continue
+  if (!str.includes('_')) return str.toLowerCase()
+    let result = ''
+    for (let i=0; i<str.length; ++i){
```

Comment:
have u checked the performance impact on this, the reason it was written like the other way was cause it was faster